docs/details/dashboards/dashboard-inventory.md

```diff
@@ -31,6 +43,19 @@ Each binary (exporter, agent) running on a client will get an `agent_type` value
 To view the agents running on a service and their health status, click **OK** or **Failed** under the **Monitoring** column. Furthermore, you can also check the properties of a particular agent by clicking the <image src="../../_images/arrow-downward.ico" width="15px" aria-label="downward arrow"/> icon under the **Options** column.
 ![!image](../../_images/PMM_Inventory_Service_Agent_Properties.png)
 
+#### Node-service relationship
+
+Starting with PMM 2.40.0, you click on the link in the **Node Name** column to view the node on which a specific service is running and analyze how node-level resource utilization impacts the performance of those services.
```

Comment:
you *can* click

or

clicking on the link in the Node Name column will show ...

Reply:
Fixed.